liquidctl-completion.sh

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env bash
+
+# This is a bash completion script for liquidctl 
+# to enable this completion place this file in /etc/bash_completion.d/ (note this locatio may be system dependant_ 
+# you can also enable this manually by sourcing this file in your ~/.bashrc file `source path/to/this` 
+#
+#
+# Copyright (C) 2020-2020 Marshall Asch 
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+
+
+
+# logging method
+_e() { echo "$1" >> log; }
+
+
+_list_bus_options () {
+    liquidctl list -v | grep 'Bus:' | cut -d ':' -f 2 | sort -u
+}
+
+_list_vendor_options () {
+    liquidctl list -v | grep 'Vendor ID:' | cut -d ':' -f 2 | cut -c4-6 | sort -u
+}
+
+_list_product_options () {
+    liquidctl list -v | grep 'Product ID:' | cut -d ':' -f 2 | cut -c4-6 | sort -u
+}
+
+_list_device_options () {
+    liquidctl list | cut -d ' ' -f 3 | cut -d ':' -f 1 | sort -u
+}
+
+_list_match_options () {
+    liquidctl list | cut -d ':' -f 2 | sort -u
+}
```

Comment:
This worked surprisingly well, but I two additional suggestions:

- make everything lowercase (the comparison is insensitive anyway)
- remove parenthesis

For reference, this is the output on my system:

```
$ liquidctl --match <TAB>
1080            (experimental)  Kraken          Smart           (X42,           X72)
Device          FTW             NZXT            (V1)            X52,            
EVGA            GTX             or              X               X62             
```